pytests/repo/setup-repo.sh

```diff
@@ -65,7 +65,7 @@ cat << EOF > ${TEST_REPO_DIR}/yum.repos.d/photon-test.repo
 [photon-test]
 name=basic
 baseurl=file://${PUBLISH_PATH}
-#metalink=file://${PUBLISH_PATH}
+#metalink=file://${PUBLISH_PATH}/metalink
```

Comment:
Is this commented out from repo file?

Reply:
yes, we uncomment when we initialize for metalink test